### PR TITLE
libkomb: font-family and text-padding should not be hardcoded

### PR DESCRIPTION
**Pull request by kalenikaliaksandr**

### CODE REVIEW

File: src/mvt_sql.py [hunk 1]
```diff
@@ -182,32 +182,6 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec, extent):
     groupby = ",".join(['"%s"' % name for name in column_names])
 
     if vec == "polygon":
-        coastline_query = """select ST_AsMVTGeom(geom, ST_TileEnvelope(%s, %s, %s), %s, 64, true) as way, %s from
```
Comment (kalenikaliaksandr) [inline src/mvt_sql.py:185]: ok, I don't think merging coastline query into generic polygon query is the right move since coastline is stored as subdivided geometry and always need to be union.

File: src/mvt_sql.py [hunk 2]
```diff
@@ -231,9 +205,9 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec, extent):
             minzoom,
             x,
             y,
-            pixel_size_at_zoom(maxzoom, pxtolerance) ** 2,
+            pixel_size_at_zoom(maxzoom, 1) ** 2,
```
Comment (kalenikaliaksandr) [inline src/mvt_sql.py:208]: while I was trying to figure out why 4th zoom tile size is near 1.5mb I discovered that most of the features are natural=water polygons which looks like points or completely not visible on 4th zoom so I think the threshold for area should be higher.

Comment (Komzpa) [reply src/mvt_sql.py:208]: I believe that useful way will be:
 - generate polygons via fast way
 - check that it is < 500 kb
 - if it is larger, re-run with buffer thing

File: src/mvt_sql.py [hunk 3]
```diff
@@ -431,7 +433,10 @@ def komap_mvt_sql(options, style):
                     raise exception 'invalid tile coordinate (%, %, %)', z, x, y;
             end case;
 
-            insert into basemap_mvts(tile_z, tile_x, tile_y, mvt) values (z, x, y, mvt);
+            insert into basemap_mvts(tile_z, tile_x, tile_y, mvt, render_time, updated_at, dirty)
```
Comment (Komzpa) [inline src/mvt_sql.py:436]: please make it a separate PR

File: src/mvt_sql.py [hunk 4]
```diff
@@ -389,10 +389,12 @@ def komap_mvt_sql(options, style):
         as $$
         declare
             mvt bytea;
+            dirty boolean;
+            t timestamp with time zone := clock_timestamp();
         begin
-            select basemap_mvts.mvt into mvt from basemap_mvts where basemap_mvts.tile_z = z and basemap_mvts.tile_x = x and basemap_mvts.tile_y = y;
+            select basemap_mvts.mvt, basemap_mvts.dirty into mvt, dirty from basemap_mvts where basemap_mvts.tile_z = z and basemap_mvts.tile_x = x and basemap_mvts.tile_y = y;
```
Comment (Komzpa) [inline src/mvt_sql.py:395]: take a lock
